dispatcher/backend/src/routes/schedules/schedule.py

```diff
@@ -204,6 +205,26 @@ def delete(self, schedule_name: str, token: AccessToken.Payload):
         return Response(status=HTTPStatus.NO_CONTENT)
 
 
+class ScheduleImageNames(BaseRoute):
+    rule = "/<string:schedule_name>/image-names"
+    name = "schedule-images"
+    methods = ["GET"]
+
+    @authenticate
+    @require_perm("schedules", "update")
+    def get(self, schedule_name: str, token: AccessToken.Payload):
+        """ proxy list of tags from docker hub """
+        request_args = request.args.to_dict()
+        hub_name = request_args.get("hub_name")
+        try:
+            data = requests.get(
```

Comment:
We should get that and just return a list of the tag names. There is no point in forwarding all that